File: .vim/template/template_test.cpp
#include <gtest/gtest.h>

#include "<+TEST TARGET+>.h"

class SomeTest : public ::testing::Test {
 protected:
  virtual void SetUp() {

  }

  virtual void TearDown() {

  }
};

TEST_F(SomeTest, test1) {
  <+CURSOR+>
}

File: .vim/template/template_test.py
#!/usr/bin/env python2
# -*- coding: utf-8 -*-

import unittest

import <+TEST TARGET+>


class SomeTest(unittest.TestCase):

    def setUp(self):
        pass

    def tearDown(self):
        pass

    def test_something(self):
        <+CURSOR+>


if __name__ == '__main__':
    unittest.main()
